### PR TITLE
chore: improve test coverage

### PR DESCRIPTION
**Pull request by edurne85**
Refactor code to eliminate redundant checks and enhance test coverage for some corner cases.

### CODE REVIEW

File: src/Creature/Feats/builder/helpers.ts [hunk 1]
```diff
@@ -106,16 +106,14 @@ function applyPendingReqs(allFeats: { [key in Feat]: FeatSpec }): void {
...
-            _feat(
-                key,
-                allOf(
-                    featSpec.requirements,
-                    ...pending.map(f => new CharacterFeatRequirement(f)),
-                ),
-                featSpec.flags,
-            );
-        }
+        _feat(
```
[the remark this answers is not in the document]
Comment (edurne85) [reply src/Creature/Feats/builder/helpers.ts:107]: Covered by type guards and type definition: `allFeats` must include all `Feat` keys, and `checkFeatKey` asserts that the current key is valid.

File: src/Bonus/MultiBonus.ts [hunk 2]
```diff
@@ -83,9 +83,7 @@ class MultiBonus implements FullMultiBonus, JsonExportable<MultiBonusBuilder> {
     export(): JsonCompatible<MultiBonusBuilder> {
         const obj: MultiBonusBuilder = {};
         Object.keys(BonusType).forEach(type => {
-            if (this[type as BonusType]) {
-                obj[type as BonusType] = this[type as BonusType].export();
-            }
+            obj[type as BonusType] = this[type as BonusType].export();
         });
```
[the remark this answers is not in the document]
Comment (edurne85) [reply src/Bonus/MultiBonus.ts:87]: `implements FullMultiBonus` ensures the object will provide properties for all `BonusType` keys.
